modules/custom/movie_controller/src/Controller/MovieController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Drupal\movie_controller\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Database\Database;
+
+class MovieController {
+
+
+  
+public function movie() {
+      
+  
+  $db = Database::getConnection();
```

Comment:
Napisala sam ti nekoliko puta da koristis DependencyInjection i EntityQuery i EntityTypeManager.

Reply:
Bez dependency injectiona ne mozes da koristis kontroler, to je "must have". Nije dobra praksa ne koristiti je.

Reply:
Takodje, rekla sam nekoliko puta da razdvojis u funkcije odredjene operacije i da nazoves varijable imaju smisla. Semanticki ti nista nije ispravno.